test/e2e/util/velero/velero_utils.go

```diff
@@ -797,3 +807,29 @@ func GetResticRepositories(ctx context.Context, veleroNamespace, targetNamespace
 
 	return common.GetListBy2Pipes(ctx, *CmdLine1, *CmdLine2, *CmdLine3)
 }
+
+func GetSnapshotCheckPoint(client TestClient, VeleroCfg VerleroConfig, expectCount int, namespaceBackedUp, backupName string, kibishiiPodNameList []string) (SnapshotCheckPoint, error) {
+	var snapshotCheckPoint SnapshotCheckPoint
+	snapshotCheckPoint.ExpectCount = expectCount
+	snapshotCheckPoint.NamespaceBackedUp = namespaceBackedUp
+	snapshotCheckPoint.PodName = kibishiiPodNameList
+	if strings.EqualFold(VeleroCfg.Features, "EnableCSI") {
+		snapshotCheckPoint.EnableCSI = true
+		if err := util.CheckVolumeSnapshotCR(client, kibishiiPodNameList, namespaceBackedUp, backupName); err != nil {
+			return snapshotCheckPoint, errors.Wrapf(err, "Fail to get Azure CSI snapshot content")
```

Comment:
Looks like this is a common function for all cloud provider. May need to remove Azure from log information.